libraries/core_libs/network/include/network/tarcap/stats/max_stats.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "json/value.h"
+#include "network/tarcap/stats/packet_stats.hpp"
+
+namespace taraxa::network::tarcap {
+
+/**
+ * @brief MAx stats data holder class
+ */
+class MaxStats {
+ public:
+  PacketStats max_count_stats_;
+  PacketStats max_size_stats_;
+  PacketStats max_processing_duration_stats_;
+  PacketStats max_tp_wait_time_stats_;
```

Comment:
because it is accessed by other class when comparing some variable and as it is simple data holder class it is public so I dont need to create 4 getters and 4 setters

Reply:
oh it is not used like it anymore, it was before - good point